apps/api/v2/src/modules/slots/slots-2024-09-04/services/slots.service.ts

```diff
@@ -147,13 +149,19 @@ export class SlotsService_2024_09_04 {
     }
 
     const nonSeatedEventAlreadyBooked = !eventType.seatsPerTimeSlot && booking;
-    if (nonSeatedEventAlreadyBooked) {
+    const isRoundRobinEvent = eventType.schedulingType === SchedulingType.ROUND_ROBIN;
```

Comment:
Can we add e2e tests in `apps/api/v2/src/modules/slots/slots-2024-09-04/controllers/e2e/team-event-type-slots.controller.e2e-spec.ts` testing
1. slot reservation for round robin where there are only non-fixed hosts
2. slot reservation for round robin where there is at least 1 fixed host